package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "fiware-sth-rest-api",
```

Comment:
Hey! Yeah, the package.json file is a descriptor file for a Node.js application or module which, amongst many other things, describes the dependencies of any other modules. It is located in the root of the application or module and it is basically used to automatically import the needed modules using https://www.npmjs.com/ (executing "npm install" in the same directory where the package.json is located).

Regarding the package/namespace issue, there is not such a mechanism in Javascript. I mean, to explicitly declare the package or namespace of a file as there is in Java. At least not in ECMAScript 5 (the current version of Javascript), since in ECMAScript 6 a new "module" feature will be included: http://wiki.ecmascript.org/doku.php?id=harmony:modules Until then, Javascript uses directories for the "namespacing". In this same sense, in Node.js there is a require() function to import components from other files or modules which have to be exposed by them in the "module.exports" object of those files ;)